h5c/vic/src/vic-webapp/src/app/container-view/container-view.template.html

```diff
@@ -1,4 +1,30 @@
-<clr-datagrid (clrDgRefresh)="refreshGrid($event)" [clrDgLoading]="isDgLoading">
+<div class="col-xs-12 mb-2" *ngIf="errorObj">
+    <div class="row">
+      <div class="col-xs-12">
+        <clr-alert [clrAlertType]="'danger'"
+                  [clrAlertClosable]="false">
+          <clr-alert-item *ngIf="errorObj.type === 'ssl_cert'">
+            <span class="alert-text">
+              <!-- TODO: i18n -->
+              Failed to verify the VIC machine server at endpoint https://{{ errorObj.payload }}:8443. Your browser may be blocking the connection because of the untrusted SSL certificate.
+            </span>
+            <div class="alert-actions">
+              <a href="https://{{ errorObj.payload }}:8443/container/version" target="_blank">Fix</a>
+              <a href="javascript://" (click)="checkVicMachineServer()">Refresh</a>
```

Comment:
Can you put a '|' between Fix and Refresh? In the browser it isn't obvious they are two separate links.

Fix | Refresh